integrationtest/wildfly/src/test/java/org/hibernate/search/test/integration/arquillian/LoggingConfigurator.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.test.integration.arquillian;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+
+import org.jboss.arquillian.config.descriptor.api.ArquillianDescriptor;
+import org.jboss.arquillian.container.spi.event.container.AfterStart;
+import org.jboss.arquillian.container.spi.event.container.BeforeStop;
+import org.jboss.arquillian.core.api.Instance;
+import org.jboss.arquillian.core.api.annotation.Inject;
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.dmr.ModelNode;
+
+import static org.jboss.as.controller.client.helpers.ClientConstants.ADD;
+import static org.jboss.as.controller.client.helpers.ClientConstants.OP;
+import static org.jboss.as.controller.client.helpers.ClientConstants.OP_ADDR;
+import static org.jboss.as.controller.client.helpers.ClientConstants.OUTCOME;
+import static org.jboss.as.controller.client.helpers.ClientConstants.REMOVE_OPERATION;
+import static org.jboss.as.controller.client.helpers.ClientConstants.SUCCESS;
+
+/**
+ * Configures WildFly so that the org.hibernate.search logger exists for the DEBUG level.
+ */
+public class LoggingConfigurator {
+
+	@Inject
+	private Instance<ManagementClient> managementClient;
+
+	private Logger logger = Logger.getLogger(LoggingConfigurator.class.getName());
```

Comment:
It's stange that checkstyle didn't notice it, but you should use spaces after opening parentheses and before closing ones.
It's rather dull to fix that manually, so you can use the codestyles for IDEA and Eclipse from there: https://github.com/hibernate/hibernate-ide-codestyles